DEV.md

```diff
@@ -0,0 +1,41 @@
+# Dev environment
+
+
+Clone the repo and install dependencies:
+```
+$ git clone git@github.com:GaloyMoney/galoy.git
+$ cd galoy
+$ yarn install
+```
+
+We use [direnv](https://direnv.net) to load environment variables needed for running the integration tests.
+```
+$ direnv allow
+direnv reload
+direnv: direnv: loading ~/projects/GaloyMoney/galoy/.envrc
+(...)
+```
+
+To execute the test suite runtime dependencies must be running.
+```
+$ make start-deps
+docker-compose up -d
```

Comment:
missing $

Reply:
No its output

Reply:
ok

Reply:
Make by default echos the commands its running back to the shell